tests/queries/0_stateless/01278_min_insert_block_size_rows_for_materialized_views.sh

```diff
@@ -53,10 +53,10 @@ EOL
 
 echo "create table null_01278 as data_01278 Engine=Null();" | execute
 for i in $(seq 1 $TEST_01278_PARTS); do
-    echo "create table part_01278_$i as data_01278 Engine=Buffer(currentDatabase(), null_01278, 1, 86400, 86400, 1e5, 1e6, 10e6, 100e6);"
+    echo "create table part_01278_$i as data_01278 Engine=Buffer('$CLICKHOUSE_DATABASE', null_01278, 1, 86400, 86400, 1e5, 1e6, 10e6, 100e6);"
```

Comment:
BTW this looks like some intermediate state? (i.e. during developing currentDatabase() was broken at some point, but fixed later?)

Reply:
I have to investigate, but for some reason the `currentDatabase()` doesn't work in some/all shell-tests, leading to very unexpected behavior: probably it's some real problems in the code which cannot properly interpret function vs literal.